packages/api/src/routes/devices/internal-user.ts

```diff
@@ -53,7 +53,7 @@ router.post(
       });
       await sendProviderDisconnected(newConnectedUser, disconnectedProviders);
     }
-    return res.status(status.OK).json({});
+    return res.status(status.OK);
```

Comment:
`.status()` doesn't send the response, we need either `.sendStatus()` or `.status().json()`